app/resources/en/translation/screeners/startapply.json

```diff
@@ -4,5 +4,5 @@
   "introduction": "Based on the information you&rsquo;ve given, you have everything you need to make your application for probate online.",
   "paragraph1": "You will need an email address to create your account.",
   "paragraph2": "You must use your own email address even if someone helps with your application. That&rsquo;s because we can&rsquo;t discuss details with anyone but the person making the application.",
-  "paragraph3": "Already registered? <a href=\"tasklist\" target=\"_self\">Log in.</a>"
+  "paragraph3": "Already registered? <a href=\"task-list\" target=\"_self\">Log in.</a>"
```

Comment:
Revert this please, I opened a ticket just for amending these links.

Reply:
done